polaris.shopify.com/content/components/choice-list.md

```diff
@@ -108,86 +109,41 @@ Shipping Options
 
 <!-- end -->
 
-- Not use colons
+### Not use colons
 
 <!-- usageblock -->
 
 #### Do
 
 If the customer abandons their checkout, send them an email reminder to complete their order
 
-<p>
-  <label><input type="radio" name="foo"> Never</label><br />
-  <label><input type="radio" name="foo"> 6 hours later</label><br />
-  <label><input type="radio" name="foo"> 24 hours later</label>
-</p>
-
 #### Don’t
 
 If the customer abandons their checkout, send them an email reminder to complete their order:
 
-<p>
-  <label><input type="radio" name="bar"> Never</label><br />
-  <label><input type="radio" name="bar"> 6 hours later</label><br />
-  <label><input type="radio" name="bar"> 24 hours later</label>
-</p>
 <!-- end -->
 
 ### List choices
 
 Every item in a choice list should:
 
 - Start with a capital letter
-
-<!-- usageblock -->
-
-#### Do
-
-- Option 1
-- Option 2
-- Option 3
-
-#### Don’t
-
-- option 1
-- option 2
-- option 3
-
-<!-- end -->
-
 - Not use commas or semicolons at the end of each line
-
-<!-- usageblock -->
-
-#### Do
-
-- Red
-- Yellow
-- Blue
-
-#### Don’t
-
-- Red;
-- Yellow;
-- Blue.
-
-<!-- end -->
-
 - Be written in sentence case (the first word capitalized, the rest lowercase)
 
 <!-- usageblock -->
 
 #### Do
 
-- Item one
-- Item two
-- Item three
+- List item 1
+- List item 2
+- List item 3
 
 #### Don’t
 
-- Item One
-- Item Two
-- Item Three
+- List Item One;
+- List Item Two;
+- List Item Three.
 
```

Comment:
You added punctuation here. To me, it confuses what the example is about.

Reply:
Fair. I updated this but I do think it should be one example instead of three.